src/pages/on-chain-governance/shared/localStorageHelper.ts

```diff
@@ -0,0 +1,8 @@
+export function getLocalStorageItem(key) {
```

Comment:
let's accept `connection` as optional param, then move all the logic like
```
    connection in getLocalStorageItem("rememberMasterNodeId")
      ? getLocalStorageItem("rememberMasterNodeId")[connection]
      : RememberMasterNodeId.Yes;
```
into here